media/css/firefox/family-nav.less

```diff
@@ -9,30 +9,76 @@
 
 // mixin for sprite images
 .fxfamilynav-sprite(@top) {
-    background-image: url(/media/img/firefox/family/nav-sprite.png);
+    background-image: url(/media/img/firefox/family/nav-sprite5.png);
     background-position: 0 @top;
 
     @media (-webkit-min-device-pixel-ratio: 1.5),
        (                min-resolution: 1.5dppx),
        (                min-resolution: 144dpi) {
 
-        background-image: url(/media/img/firefox/family/nav-sprite-high-res.png);
-        .background-size(146px 273px);
+        background-image: url(/media/img/firefox/family/nav-sprite-high-res5.png);
+        .background-size(146px 297px);
     }
 }
 
-// adding this just to cache bust an image for bug 1139003
-.foo-bar {
-    font-size: inherit;
+// big left hand logo
+.fxfamilynav-logo {
+    float: left;
+    width: 140px;
+    height: 146px;
+    margin: 8px @baseLine 0 (@baseLine / 2);
+    .fxfamilynav-sprite(0);
+    .image-replaced();
+    .transition(transform 0.3s ease);
```

Comment:
Think this might not work in Safari quite right, since `transform` will need to be `-webkit-transform`